src/main/java/frc/robot/commands/shooter/ShootSpeaker.java

```diff
@@ -4,24 +4,25 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.constants.Constants;
 import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.SwerveDrivetrain;
 
 public class ShootSpeaker extends Command {
 
     private final Shooter shooter;
     private final Timer timer = new Timer();
     private boolean activated = false;
-    private double speed;
-    public ShootSpeaker(Shooter shooter) {
+    private SwerveDrivetrain drivetrain;
+    public ShootSpeaker(Shooter shooter, SwerveDrivetrain drivetrain) {
         this.shooter = shooter;
+        this.drivetrain = drivetrain;
         addRequirements(shooter);
     }
 
     @Override
     public void initialize() {
         timer.reset();
         activated = true;
-        shooter.setShooterMotorLeftSpeed(Constants.SHOOTER_MOTOR_LEFT_SPEED);
-        shooter.setShooterMotorRightSpeed(Constants.SHOOTER_MOTOR_RIGHT_SPEED);
+        shooter.setShooterSpeeds(drivetrain);
```

Comment:
Commands for setting speed on the motors have to be executed every cycle or else the watchdog is going to disable the robot.